src/main/kotlin/org/meogo/global/utill/FcmUtil.kt

```
@@ -15,38 +15,37 @@ class FcmUtil {
     private val firebase: FirebaseMessaging
         get() = FirebaseMessaging.getInstance()
 
-    fun sendMessage(fcmToken: List<String>, title: String, message: String) {
-        val message = messageSetting(fcmToken, title, message)
+    fun sendMessage(fcmToken: List<String>, title: String, content: String) {
+        val message = messageSetting(fcmToken, title, content)
 
         try {
             firebase.sendMulticastAsync(message)
         } catch (e: FirebaseMessagingException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)